src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,98 @@
+import "./css/Auth.css";
+import "./css/LoginPage.css";
+import { Link } from "react-router-dom";
+import { FieldName } from "../types/field";
+import useAuthStore from "../stores/useAuthStore";
+import Button from "../components/Button";
+import Label from "../components/Label";
+import Textfield from "../components/Textfield";
+import logo from "../img/logo.png";
+import kakakoIcon from "../img/kakao.png";
+import googleIcon from "../img/google.png";
+import { useEffect } from "react";
+
+const LoginPage = () => {
+  const { email, password, setField, validateField, resetFields } =
+    useAuthStore();
+
+  /* 유효성 체크 */
+  const updateValidate = (name: string, value: string) => {
+    validateField(name as FieldName, value);
+  };
+
+  /* 텍스트필드 데이터 세팅과 유효성 체크 */
+  const onChangeTextfield = (name: string, value: string) => {
+    let trimVal = value.trim();
+    setField(name as FieldName, trimVal);
+    updateValidate(name, trimVal);
+  };
+
+  const disableLoginButton = () => {
+    return !(email.validInfo.isValid && password.validInfo.isValid);
+  };
+
+  useEffect(() => {
+    resetFields();
+  }, [resetFields]);
```

Comment:
```suggestion
useEffect(() => {
  resetFields();
}, []); 
```

초기화 용도라면 의존성을 빈 배열로 변경하는것을 권장합니다.
이유는, 해당 로직은 페이지 진입 시 첫 한 번만 실행되면 되고,  외부 의존성이 없는 순수 함수이기때문입니다.